packages/core/parcel-bundler/src/assets/CSSAsset.js

```diff
@@ -1,17 +1,26 @@
 const Asset = require('../Asset');
 const postcss = require('postcss');
+const path = require('path');
+const fs = require('@parcel/fs');
 const valueParser = require('postcss-value-parser');
 const postcssTransform = require('../transforms/postcss');
 const CssSyntaxError = require('postcss/lib/css-syntax-error');
+const SourceMap = require('../SourceMap');
+const logger = require('@parcel/logger');
 
 const URL_RE = /url\s*\("?(?![a-z]+:)/;
 const IMPORT_RE = /@import/;
 const PROTOCOL_RE = /^[a-z]+:/;
+const SOURCEMAP_RE = /\/\*\s*[@#]\s*sourceMappingURL\s*=\s*([^\s]+)\s*\*\//;
```

Comment:
Well, not quite:
CSS: `/* #sourceMappingURL= */`
JS: `// #sourceMappingURL= `

So this would be possible:
`const SOURCEMAP_RE = new RegExp("\/\*"+ common-part + "\s*\*\/")`